tests/Resources/Page/OCPDashboard/OperatorHub/OperatorHub.robot

```diff
@@ -3,43 +3,49 @@ Library     SeleniumLibrary
 
 *** Keywords ***
 Install Operator
-    [Arguments]    ${operator}
+    [Arguments]    ${operator}        ${catalog}=operators
```

Comment:
For now i will resolve this conversation..and i kept it same since when we work with wither type of operator we will specifically give from while using this keyword